src/UI/class-recommended-widget.php

```diff
@@ -326,21 +361,36 @@ private function get_boost_params(): array {
 	private function api_key_and_secret_are_populated(): bool {
 		$options = $this->parsely->get_options();
 
-		// No options are saved, so API key is not available.
-		if ( ! is_array( $options ) ) {
-			return false;
-		}
```

Comment:
`$options` always be `array` and have keys defined with at least default values.